### ISSUE TITLE
script ran too long error

### ISSUE THREAD

**Issue by gaveer** [joaoc-pires/wow-addon-conceal#12]
I am getting this error on every combat with my **rogue**

```
16x Conceal/Conceal.lua:615: script ran too long
[string "@Conceal/Conceal.lua"]:615: in function `FadeOut'
[string "@Conceal/Conceal.lua"]:829: in function `HideElements'
[string "@Conceal/Conceal.lua"]:843: in function `?'
[string "@Ace3/CallbackHandler-1.0-8/CallbackHandler-1.0.lua"]:109: in function <...Ons/Ace3/CallbackHandler-1.0/CallbackHandler-1.0.lua:109>
[string "=[C]"]: ?
[string "@Ace3/CallbackHandler-1.0-8/CallbackHandler-1.0.lua"]:19: in function <...Ons/Ace3/CallbackHandler-1.0/CallbackHandler-1.0.lua:15>
[string "@Ace3/CallbackHandler-1.0-8/CallbackHandler-1.0.lua"]:54: in function `Fire'
[string "@Ace3/AceEvent-3.0-4/AceEvent-3.0.lua"]:120: in function <Ace3/AceEvent-3.0/AceEvent-3.0.lua:119>

Locals:
self = <table> {
 DidEnterCombat = <function> defined @Conceal/Conceal.lua:849
 SetDefaultModuleLibraries = <function> defined @Ace3/AceAddon-3.0/AceAddon-3.0.lua:363
 Enable = <function> defined @Ace3/AceAddon-3.0/AceAddon-3.0.lua:290
 RegisterChatCommand = <function> defined @Ace3/AceConsole-3.0/AceConsole-3.0.lua:81
 EnableModule = <function> defined @Ace3/AceAddon-3.0/AceAddon-3.0.lua:328
 modules = <table> {
 }
 CancelTimer = <function> defined @Ace3/AceTimer-3.0/AceTimer-3.0.lua:147
 IterateEmbeds = <function> defined @Ace3/AceAddon-3.0/AceAddon-3.0.lua:438
 optionsFrame = Frame {
 }
 defaultModuleLibraries = <table> {
 }
 TimeLeft = <function> defined @Ace3/AceTimer-3.0/AceTimer-3.0.lua:172
 HideElements = <function> defined @Conceal/Conceal.lua:794
 UnregisterChatCommand = <function> defined @Ace3/AceConsole-3.0/AceConsole-3.0.lua:107
 SetStatus = <function> defined @Conceal/Conceal.lua:923
 ScheduleRepeatingTimer = <function> defined @Ace3/AceTimer-3.0/AceTimer-3.0.lua:129
 CancelAllTimers = <function> defined @Ace3/AceTimer-3.0/AceTimer-3.0.lua:160
 Printf = <function> defined @Ace3/AceConsole-3.0/AceConsole-3.0.lua:65
 baseName = "Conceal"
 SetDefaultModulePrototype = <function> defined @Ace3/AceAddon-3.0/AceAddon-3.0.lua:405
 GetName = <function> defined @Ace3/AceAddon-3.0/AceAddon-3.0.lua:275
 GetSlider = <function> defined @Conceal/Conceal.lua:967
 defaultModuleState = true
 name = "Conceal"
 GetStatus = <function> defined @Conceal/Conceal.lua:889
 IsEnabled = <function> defined @Ace3/AceAddon-3.0/AceAddon-3.0.lua:447
 orderedModules = <table> {
 }
 ScheduleTimer = <function> defined @Ace3/AceTimer-3.0/AceTimer-3.0.lua:94
 DisableModule = <function> defined @Ace3/AceAddon-3.0/AceAddon-3.0.lua:346
 ShowMouseOverElements = <function> defined @Conceal/Conceal.lua:652
 FadeOut = <function> defined @Conceal/Conceal.lua:594
 Disable = <function> defined @Ace3/AceAddon-3.0/AceAddon-3.0.lua:310
 loadConfig = <function> defined @Conceal/Conceal.lua:874
 RegisterMessage = <function> defined @Ace3/CallbackHandler-1.0/CallbackHandler-1.0.lua:80
 GetArgs = <function> defined @Ace3/AceConsole-3.0/AceConsole-3.0.lua:140
 Print = <function> defined @Ace3/AceConsole-3.0/AceConsole-3.0.lua:50
 TargetChanged = <function> defined @Conceal/Conceal.lua:839
 ProfileHandler = <function> defined @Conceal/Conceal.lua:869
 HideGcdFlash = <function> defined @Conceal/Conceal.lua:861
 DidExitCombat = <function> defined @Conceal/Conceal.lua:854
 IsModule = <function> defined @Ace3/AceAddon-3.0/AceAddon-3.0.lua:464
 db = <table> {
 }
 SetDefaultModuleState = <function> defined @Ace3/AceAddon-3.0/AceAddon-3.0.lua:383
 ShowCombatElements = <function> defined @Conceal/Conceal.lua:621
 FadeIn = <function> defined @Conceal/Conceal.lua:569
 SetEnabledState = <function> defined @Ace3/AceAddon-3.0/AceAddon-3.0.lua:420
 isHealthBelowThreshold = <function> defined @Conceal/Conceal.lua:557
 enabledState = true
 OnInitialize = <function> defined @Conceal/Conceal.lua:527
 RegisterEvent = <function> defined @Ace3/CallbackHandler-1.0/CallbackHandler-1.0.lua:80
 UnregisterAllMessages = <function> defined @Ace3/CallbackHandler-1.0/CallbackHandler-1.0.lua:164
 UnregisterAllEvents = <function> defined @Ace3/CallbackHandler-1.0/CallbackHandler-1.0.lua:164
 SendMessage = <function> defined @Ace3/CallbackHandler-1.0/CallbackHandler-1.0.lua:49
 ```

**Comment by gaveer** [comment]
```
-- Replace this line in the OnInitialize function
C_Timer.NewTicker(0.10, function() ... end)

-- With a higher interval, for example:
C_Timer.NewTicker(1.0, function() ... end)
```

**Comment by joaoc-pires** [comment]
Hey @gaveer, sorry I didn't reply at the time. Your suggest fix has been implemented and deployed in 1.6.0
